### DISCUSSION TITLE
Ex13, total boundary current as the boundary condition?

### DISCUSSION THREAD

**Discussion by gatling-nrl** [kinnala/scikit-fem#770] [Q&A]
In [Ex 13](https://github.com/kinnala/scikit-fem/blob/bdaf7126b0317be0481568556244fa2bd5daee0d/docs/examples/ex13.py#L66) the current into the domain through the boundary is calculated.

Instead of specifying the voltage everywhere on that part of the boundary like ex 13, is it possible to specify the total current through that part of the boundary as the boundary condition?

**Comment by kinnala** [comment]
Something like in https://github.com/kinnala/scikit-fem/blob/master/docs/examples/ex05.py ?

**Reply by kinnala** [reply]
Note that @gdmcbain had some doubts on the validity of such boundary condition in #751.
